backend/src/main/java/de/neuefische/backend/service/CovidTestService.java

```diff
@@ -44,5 +44,14 @@ public CovidTest addCovidTest(CovidTestDto covidTestDto) {
             return covidTestRepo.save(covidTest);
         }
     }
-    
+
```

Comment:
```suggestion
import static CovidTestMapper.*
```
makes the code further down more readable